npm/dz5/task3.js

```diff
@@ -0,0 +1,6 @@
+let a = 5;
+let b = 1;
+
+for(; b<11; b++){
```

Comment:
Буде зрозуміліше, якщо написати:
```suggestion
for(let b = 1; b<11; b++){
```